admin-dev/themes/new-theme/js/components/navbar-transition-handler.js

```diff
@@ -47,7 +50,10 @@ function NavbarTransitionHandler($navBar, $mainMenu, endTransitionEvent, $body)
 
     this.$navBar.removeEventListener(this.endTransitionEvent, this.showNavBarContent);
     const isSidebarClosed = this.$body.hasClass('page-sidebar-closed');
-    this.$mainMenu.toggleClass('sidebar-closed', isSidebarClosed);
+
+    if ($(window).width() > MAX_MOBILE_WIDTH) {
+      this.$mainMenu.toggleClass('sidebar-closed', isSidebarClosed);
```

Comment:
I'm a bit picky, but don't you think that we should use a class map for every edited component in the new theme?

Reply:
I guess it doesn't have too much sense if it's not a real selector, also, class maps are not used by anyone, really nitpick to me